actionpack/test/abstract/translation_test.rb

```diff
@@ -47,16 +47,6 @@ def test_action_controller_base_responds_to_l
         assert_respond_to @controller, :l
       end
 
-      def test_raises_missing_translation_message_with_raise_config_option
-        AbstractController::Translation.raise_on_missing_translations = true
-
-        assert_raise(I18n::MissingTranslationData) do
```

Comment:
I think we should keep this test but move to the railties tests to make sure that setting `config.i18n. raise_on_missing_translations` affects the controller's version of `t`.